### ISSUE TITLE
Support for -R / --recursive

### ISSUE THREAD

**Issue by athityakumar** [athityakumar/colorls#29]
This flag option can be used to recursively print all files inside sub-directories.

Side note : The existing `-r / --report` flag can be changed to `-i / --info`.

**Comment by athityakumar** [comment]
Update : As symlink support is being added in `-r` flag, this option has been updated to `-R`.

**Comment by athityakumar** [comment]
Fixed with `-t` / `--tree` flag.

**Comment by kaustubhhiware** [comment]
@athityakumar Can you correct your [second comment](https://github.com/athityakumar/colorls/issues/29#issuecomment-314328043) above as follows  
```diff
- is being added in `-r` flag
+ is being added in `-t` flag
```

This would be convenient in the future and remove any misunderstandings (and reduce some emails :sweat_smile: ) later. This comment will be removed later.

**Comment by athityakumar** [comment]
Oops, sorry for the confusion. Thanks for pointing out, @kaustubhhiware. This is what I meant -

- Initially, I wanted a `-r` / `--recursive` flag that opens all sub-directories and shows files present in them.
- Then, a request came for symlinks with `-r` flag. Hence, I renamed `--recursive` to `-R`.
- However, symlink support has been added with `-l` flag itself. And, `--recursive` / `-R` functionality has been added with `-t` / `--tree` flag. Hence, closed this issue. 😄
